values.yaml

```diff
@@ -5,8 +5,8 @@
 replicaCount: 1
 
 image:
-  repository: daichi703n/elastalert
-  tag: 0.2.1-dev
+  repository: praecoapp/elastalert-server
+  tag: 20230219
```

Comment:
```suggestion
  tag: "20230219"
```

Reply:
Without quote, tag will be handled as number and fail to pull. )`image tag "praecoapp/elastalert-server:2.0230219e+07"`)

```
kubectl describe pod/elastalert-79fcd5cc4d-5m4d6
---
...
  Warning  Failed                  4m53s (x10 over 6m30s)  kubelet                  Error: InvalidImageName
  Warning  InspectFailed           86s (x25 over 6m30s)    kubelet                  Failed to apply default image tag "praecoapp/elastalert-server:2.0230219e+07": couldn't parse image reference "praecoapp/elastalert-server:2.0230219e+07": invalid reference format
```